src/components/NavigationMenu/SideNavigationMenu.tsx

```diff
@@ -50,7 +61,10 @@ const SideMenuItem = ({ href, hrefAs, label, icon, active = false, disabled = fa
             fontWeight="normal"
             leftIcon={icon ? cloneElement(icon, { width: '18px', 'aria-hidden': true }) : null}
           >
-            {label}
+            <Stack direction="row" alignItems="center">
+              <>{label}</>
```

Comment:
```suggestion
              <span flex="1">{label}</span>
```

It bothers me a little that the labels aren't pushed to the right, wyt? 
Right now they are a little misaligned with each other.